question-service/README.md

```diff
@@ -0,0 +1,10 @@
+Dependencies: Python 12
+
+## Setup
+1. In the `question-service` directory, run `python -m venv .venv` to create a virtual environment.
+2. Run `source .venv/bin/activate` to activate the virtual environment.
```

Comment:
Btw, this is different for Windows, supposed to just run .venv/Scripts/activate, apart from that, its the same